src/custom/pages/Claim/styled.ts

```diff
@@ -1832,7 +1832,16 @@ export const BannerExplainer = styled.div`
   }
 
   > svg {
-    .stop1 { stop-color: ${({ theme }) => theme.white}}
-    .stop2 { stop-color: ${({ theme }) => theme.white}; stop-opacity: 0.8;}
-    .stop3 { stop-color: ${({ theme }) => theme.white}; stop-opacity: 0;}
+    .stop1 {
```

Comment:
In principle, this banner seems too general to be in the Claim page. Maybe should be a component or something? 

Since you are using it also in the profile. 
Normally, if a component is defined in a page, is because is too specific for being in the `components` dir.

Reply:
Yes, agreed. Had the exact same thoughts. When working on it, I initially thought I could get away with simply re-using the same Component. But quickly realized I need more generic `Card` components, including a prop to define 'banner' like features. I plan to tackle this outside this PR and will create a ticket specifically for this.